app/controllers/provider/admin/account/payment_gateways/braintree_blue_controller.rb

```diff
@@ -42,14 +41,12 @@ def hosted_success
       if braintree_blue_crypt.update_user(braintree_response)
         redirect_to_success
       else
-        flash[:notice] = 'Credit Card details could not be stored.'
+        flash.now[:warning] = t('.credit_card_error')
```

Comment:
Same as the emails controller - why is this `warning`, while all others errors use `danger`.

I am suspecting that this is because the original flash was `notice`, and not `error`, but I think this was a mistake.